### DISCUSSION TITLE
Vercel deployment failed with error Module not found.

### DISCUSSION THREAD

**Discussion by ethndotsh** [vercel/next.js#20189] [Help] [answered]
I am using the default blog 'learn Next.js' template, which I made from taking the course, and am having trouble deploying to Vercel. Here is my log:

19:51:59 | Cloning github.com/cursecodes/nextjs-blog (Branch: main, Commit: dda6f3d)
-- | --
19:51:59 | Cloning completed in 821ms
19:52:00 | Analyzing source code...
19:52:00 | Installing build runtime...
19:52:01 | Build runtime installed: 483.089ms
19:52:01 | Looking up build cache...
19:52:01 | Build cache not found
19:52:02 | Installing dependencies...
19:52:11 | > sharp@0.26.2 install /vercel/workpath0/node_modules/sharp
19:52:11 | > (node install/libvips && node install/dll-copy && prebuild-install) \|\| (node-gyp rebuild && node install/dll-copy)
19:52:11 | info sharp Using cached /vercel/.npm/_libvips/libvips-8.10.0-linux-x64.tar.br
19:52:12 | > @ampproject/toolbox-optimizer@2.7.0-alpha.1 postinstall /vercel/workpath0/node_modules/@ampproject/toolbox-optimizer
19:52:12 | > node lib/warmup.js
19:52:12 | AMP OPTIMIZER Downloaded latest AMP runtime data.
19:52:13 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/watchpack-chokidar2/node_modules/fsevents):
19:52:13 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
19:52:13 | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
19:52:13 | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
19:52:13 | added 703 packages from 472 contributors in 10.622s
19:52:13 | 75 packages are looking for funding
19:52:13 | run `npm fund` for details
19:52:13 | Running "npm run build"
19:52:13 | > learn-starter@0.1.0 build /vercel/workpath0
19:52:13 | > next build
19:52:14 | info  - Creating an optimized production build...
19:52:14 | Attention: Next.js now collects completely anonymous telemetry regarding usage.
19:52:14 | This information is used to shape Next.js' roadmap and prioritize features.
19:52:14 | You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
19:52:14 | https://nextjs.org/telemetry
19:52:16 | Failed to compile.
19:52:16 | ModuleNotFoundError: Module not found: Error: Can't resolve 'next/Head' in '/vercel/workpath0/pages'
19:52:16 | > Build error occurred
19:52:16 | Error: > Build failed because of webpack errors
19:52:16 | at build (/vercel/workpath0/node_modules/next/dist/build/index.js:15:918)
19:52:16 | at processTicksAndRejections (internal/process/task_queues.js:97:5)
19:52:16 | npm ERR! code ELIFECYCLE
19:52:16 | npm ERR! errno 1
19:52:16 | npm ERR! learn-starter@0.1.0 build: `next build`
19:52:16 | npm ERR! Exit status 1
19:52:16 | npm ERR!
19:52:16 | npm ERR! Failed at the learn-starter@0.1.0 build script.
19:52:16 | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
19:52:16 | npm ERR! A complete log of this run can be found in:
19:52:16 | npm ERR!     /vercel/.npm/_logs/2020-12-15T02_52_16_563Z-debug.log
19:52:16 | Error: Command "npm run build" exited with 1
19:52:18 | Done with "package.json"

And here is the link to my repository: https://github.com/cursecodes/nextjs-blog

**Comment by jamesmosier** [comment, marked as answer]
You have [an import](https://github.com/cursecodes/nextjs-blog/blob/dda6f3d57620e698f703e58731462f938949f592/pages/index.js#L1) that is incorrect. `import Head from 'next/Head'` should be `import Head from 'next/head'`

**Reply by ethndotsh** [reply]
Thanks!
